test-framework/db-mssql/src/main/java/org/keycloak/test/framework/database/MSSQLServerTestDatabase.java

```diff
@@ -0,0 +1,19 @@
+package org.keycloak.test.framework.database;
+
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.MSSQLServerContainer;
+
+class MSSQLServerTestDatabase extends AbstractContainerTestDatabase {
+
+    private static final String IMAGE_NAME = "mcr.microsoft.com/mssql/server:latest";
+
+    @Override
+    public JdbcDatabaseContainer<?> createContainer() {
+        return new MSSQLServerContainer<>(IMAGE_NAME);
```

Comment:
@stianst Don't we want to remove the license files from [examples](https://github.com/keycloak/keycloak/blob/main/test-framework/examples/src/test/resources/container-license-acceptance.txt) and base [test](https://github.com/keycloak/keycloak/blob/main/tests/base/src/test/resources/container-license-acceptance.txt) and rather use acceptLicense() [method](https://java.testcontainers.org/modules/databases/mssqlserver/) in here?

Reply:
I don't think we can/should accept the license on behalf of users using the framework. So, should be placed alongside tests, and not in the framework itself.